src/components/FormInput.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+
+import { Text, TextInput, TextInputProps, GestureResponderEvent, StyleSheet, View } from 'react-native';
+import {
+    useController,
+    useFormContext,
+    ControllerProps,
+    UseControllerProps
+} from 'react-hook-form';
+
+import globalStyles from '../globalStyles';
+
+/* EXTENDING PROPS TYPES TO INHERIT NAME AND RULES FROM USECONTROLLERPROPS */
+interface FormInputProps extends TextInputProps, UseControllerProps {
+    label: string
+    defaultValue?: string //ADD DEFAULT VALUE TO PROPS
+}
+
+const ControlledInput = (props: FormInputProps) => {
+
+    const formContext = useFormContext();
+    // const { formState } = formContext;
+
+    const {
+        name,
+        label,
+        rules,
+        defaultValue,
+        ...inputProps
+    } = props;
+
+    const { field } = useController({ name, label, rules, defaultValue });
+
+    const styles = StyleSheet.create({
```

Comment:
We decided to still keep stylesheets in their own styles.tsx file in a folder with the component for consistency
Do that even though your stylesheet is really small